components/raftstore/src/store/worker/pd.rs

```diff
@@ -2006,6 +2025,20 @@ where
         }
         let id = slow_score_tick_result.tick_id;
         let scheduler = self.scheduler.clone();
+
+        if factor == InspectFactor::Network {
+            let duration = std::collections::HashMap::<u64, Duration>::new();
```

Comment:
Waiting for https://github.com/tikv/tikv/pull/18798 to get the real latency

Reply:
Why not move this part into
```
    match factor {
        InspectFactor::RaftDisk => {xxx},
        InspectFactor::KvDisk => {xxx},
        InspectFactor::Network => {...}
```